packages/soci-snapshotter/soci-snapshotter.spec

```
@@ -13,13 +13,18 @@ URL: https://github.com/awslabs/soci-snapshotter
 Source0: https://github.com/awslabs/soci-snapshotter/archive/v%{gover}/soci-snapshotter-%{gover}.tar.gz
 Source1: bundled-soci-snapshotter-%{gover}.tar.gz
 Source2: bundled-cmd.tar.gz
+Source3: soci-config-toml
+# Mount for writing soci configuration
+# Generated with systemd-escape -p --suffix=mount "/etc/soci-snapshotter"
+Source100: "etc-soci\\x2dsnapshotter.mount"
 Source101: soci-snapshotter.service
 Source102: soci-snapshotter.socket
 Source1000: clarify.toml
 
 BuildRequires: %{_cross_os}glibc-devel
 BuildRequires: %{_cross_os}libz-devel
 Requires: %{name}(binaries)
+Requires: %{name}(optimized-gunzip)
```

Following up to my comment [here](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/576#discussion_r2205883876), we could add:
```suggestion
Requires: (%{name}-k8s if %{_cross_os}variant-runtime(k8s))
```

To automatically install a `k8s` subpackage containing the containerd (CRI) and k8s drop-ins when installing SOCI on a k8s variant.